src/sentry/relocation/models/relocationtransfer.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import annotations
+
+from django.db import models
+from django.utils import timezone
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import control_silo_model, region_silo_model
+from sentry.db.models.base import DefaultFieldsModel
+from sentry.db.models.fields.uuid import UUIDField
+
+
+class RelocationTransferState(models.TextChoices):
+    Request = "request"
+    Reply = "reply"
+
+
+class BaseRelocationTransfer(DefaultFieldsModel):
+    """
+    Base class for control + region relocation transfer models
+
+    Relocation transfers are used to record a retriable
+    state of a regional transfer for relocation data.
+    These models replace outbox based transfers.
+    """
+
+    __relocation_scope__ = RelocationScope.Excluded
+
+    relocation_uuid = UUIDField()
+    org_slug = models.CharField(null=False)
+    requesting_region = models.CharField(null=False)
+    exporting_region = models.CharField(null=False)
+    state = models.CharField(
+        choices=RelocationTransferState, default=RelocationTransferState.Request
+    )
+    scheduled_for = models.DateTimeField(null=True, default=timezone.now)
+    public_key = models.BinaryField(null=True)
+
+    class Meta:
+        abstract = True
+
+
+@control_silo_model
+class ControlRelocationTransfer(BaseRelocationTransfer):
+    __relocation_scope__ = RelocationScope.Excluded
+
+    class Meta:
+        app_label = "sentry"
```

Comment:
What is the `ControlRelocationTransfer` model intended to address in the spec? The region side makes sense for region resource exports, but it's not clear what ControlOutboxes we'll need to replace.

Reply:
Because regions can't talk to each other directly, when a relocation is started on the 'requesting region', it first sends the export request to control silo, so that the export request can be forwarded to the exporting region.